extras/kms/Makefile

```diff
@@ -0,0 +1,25 @@
+PHONY: tools
+tools:
+	go generate -tags tools tools/tools.go
+
+.PHONY: fmt
+fmt:
+	gofumpt -w $$(find . -name '*.go')
```

Comment:
Minor note, I've been using this:
```shell
$ grep -L -R "^\/\/ Code generated .* DO NOT EDIT\.$" --include="*.go" | xargs -n 1 gofumpt -w
```
Which ensure that generated files that conform to https://pkg.go.dev/cmd/go#hdr-Generate_Go_files_by_processing_source do not get formatted (like protobuf files). Might be a bit overkill in this repo but wanted to mention it 😁.